sample-android-app/src/main/java/com/badoo/reaktive/sample/android/MainActivity.kt

```diff
@@ -24,11 +25,11 @@ class MainActivity : AppCompatActivity() {
 
         findViewById<Button>(R.id.button).setOnClickListener {
             observable<String> { emitter ->
-                emitter.onNext("Loading...")
                 Thread.sleep(1000L)
                 emitter.onNext(SimpleDateFormat.getDateTimeInstance().format(Date()))
                 emitter.onComplete()
             }
+                .startWith("Loading...")
```

Comment:
Better to put it after 'observeOn', so only blocking observable will be affected by thread switching.

Reply:
Done